subprojects/testing-base/src/integTest/groovy/org/gradle/testing/TestExecutionBuildOperationsIntegrationTest.groovy

```diff
@@ -85,6 +89,7 @@ class TestExecutionBuildOperationsIntegrationTest extends AbstractIntegrationSpe
 
     def "emits test operations as expected for composite builds"() {
```

Comment:
The init script should support composite builds. All the included builds are known after settings has been evaluated, so they can be configured by an init script.

Reply:
Yeah.. the global init script is not necessary... it's very curious why it didn't work for me previously...